.github/workflows/helm-integration-test-latest.yml

```diff
@@ -0,0 +1,17 @@
+name: Helm Integration Test (latest)
+
+on:
```

Comment:
Can we also have a workflow_dispatch here? Because sometimes we only update the `mgmt-backend` without updating `base`. So we need to manually trigger it to run this test.

Reply:
@donch1989 Sure, I updated the workflow trigger along push